share/picongpu/tests/KHI_growthRate/include/picongpu/param/particleFilters.param

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2013-2022 Rene Widera
```

Comment:
not needed - can be removed - why do we actually have this in the KHI example?

Reply:
this file contains the selection of the particle moving up and down.